docs/tutorials/10_pvqd.ipynb

```diff
@@ -0,0 +1,563 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qiskit.primitives import Sampler, Estimator\n",
+    "from qiskit_algorithms.state_fidelities import ComputeUncompute\n",
+    "\n",
+    "# the fidelity is used to evaluate the objective: the overlap of the variational form and the trotter step\n",
+    "sampler = Sampler()\n",
+    "fidelity = ComputeUncompute(sampler)\n",
+    "\n",
+    "# the estimator is used to evaluate the observables\n",
+    "estimator = Estimator()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Since p-VQD performs a classical optimization in each timestep to determine the best parameters for the projection, we also have to specify the classical optimizer. As a first example we're using BFGS, which typically works well in statevector simulations, but later we can switch to gradient descent."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qiskit_algorithms.optimizers import L_BFGS_B\n",
+    "\n",
+    "bfgs = L_BFGS_B()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we can define p-VQD and execute it!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qiskit_algorithms.time_evolvers.pvqd import PVQD\n",
+    "\n",
+    "pvqd = PVQD(\n",
+    "    fidelity,\n",
+    "    ansatz,\n",
+    "    initial_parameters,\n",
+    "    estimator=estimator,\n",
+    "    num_timesteps=100,\n",
+    "    optimizer=bfgs\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The p-VQD implementation follows Qiskit's time evolution interface, thus we pack all information of the evolution problem into an input class: the `hamiltonian` under which we evolve the state, the `final_time` of the evolution and the observables (`aux_operators`) we keep track of."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qiskit_algorithms.time_evolvers.time_evolution_problem import TimeEvolutionProblem\n",
+    "\n",
+    "problem = TimeEvolutionProblem(hamiltonian, time=final_time, aux_operators=[hamiltonian, observable])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "And then run the algorithm!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "result = pvqd.evolve(problem)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we can have a look at the results, which are stored in a `PVQDResult` object. This class has the fields\n",
```

Comment:
Should we print(result) in the last cell - this talks about all of the fields but we cannot see anything of what actually was the result - unless its rather large when printed which is perhaps why it was not done